pom.xml

```diff
@@ -97,7 +97,7 @@
         <dependency>
             <groupId>uk.gov.ons.ctp.product</groupId>
             <artifactId>actionsvc-api</artifactId>
-            <version>10.49.22</version>
+            <version>10.49.23-SNAPSHOT</version>
```

Comment:
can probably flip this to use the release version now

Reply:
Made the change just waiting for acceptance tests to pass locally before pushing it up